lib/galaxy/datatypes/binary.py

```diff
@@ -3759,6 +3759,35 @@ def sniff(self, filename: str) -> bool:
         return Fast5Archive.sniff(self, filename)
 
 
+class Pod5(Binary):
+    """
+    Class describing a Pod5 file
```

Comment:
Can you add a link to the specification here or in the data types config file?

Reply:
Done in 385e7ab1e1f35c596b79efa7ddfffe0d4da753e4